### DISCUSSION TITLE
Is there a way to auto-activate discussions?

### DISCUSSION THREAD

**Discussion by OrangoMango** [community/community#3313] [Discussions]
Is there a way to auto-activate discussions from settings?

**Comment by PranamBhat** [comment]
You can enable or disable discussions for a repository. On GitHub, navigate to the main page of the repository. Under your repository name, click Settings. Under "Features", select Discussions.

In general case : 

Go to [https://github.com/github/feedback](https://github.com/github/feedback) and Click on "Discussions" and Click on "New Discussion".
